CMakeLists.txt

```diff
@@ -200,9 +205,11 @@ elseif(_ARCH_64 AND CMAKE_SYSTEM_PROCESSOR MATCHES "x86_64|amd64|AMD64")
   add_definitions(-D_M_X86_64=1)
   check_and_add_flag(HAVE_SSE2 -msse2)
 elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "aarch64")
+  if (NOT MSVC)
```

Comment:
The `_M_ARM` could probably be dropped, I can't see any references to it in the source.